src/collector/StatsParser.cpp

```diff
@@ -144,27 +176,57 @@ Parser::UIntInfoVector backend_uint_info = {
     { Backends|BackendFolder|Backend|BaseStats|BlobFilename|BlobBaseSize, MAX, BOFF(max_blob_base_size)   },
     { Backends|BackendFolder|Status|DefragState,                          SET, BOFF(defrag_state)         },
     { Backends|BackendFolder|Status|State,                                SET, BOFF(state)                },
+    { Backends|BackendFolder|Status|ReadOnly,                             SET, BOFF(read_only)            },
+    { Backends|BackendFolder|Status|LastStart|LastStartTvSec,             SET, BOFF(last_start_ts_sec)    },
+    { Backends|BackendFolder|Status|LastStart|LastStartTvUsec,            SET, BOFF(last_start_ts_usec)   },
     { Timestamp|TvSec,                                                    SET, NOFF(ts_sec)               },
     { Timestamp|TvUsec,                                                   SET, NOFF(ts_usec)              },
     { Procfs|Vm|La,                                                       SET, NOFF(la1)                  },
     { Procfs|Net|NetInterfaces|NetInterfaceName|Receive|Bytes,            SUM, NOFF(rx_bytes)             },
-    { Procfs|Net|NetInterfaces|NetInterfaceName|Transmit|Bytes,           SUM, NOFF(tx_bytes)             }
+    { Procfs|Net|NetInterfaces|NetInterfaceName|Transmit|Bytes,           SUM, NOFF(tx_bytes)             },
+    { Stats|StatName|Count,                                               SET, SOFF(count)                }
 };
 
 Parser::StringInfoVector backend_string_info;
 
 } // unnamed namespace
 
+StatsParser::Data::Data()
```

Comment:
initialize stat_commit in consructor initialization list. It will be default initialized (filled with zeroes)